release-notes/release-notes58.md

```diff
@@ -48,33 +42,36 @@ For information on editor resizing, see: [User interface options - Resize]({{sit
 
 {{site.productname}} 5.8 introduces the following minor new features:
 
-- changelog
+- Added the `PAGE_UP` and `PAGE_DOWN` key code constants to the `VK` API.
 
 ## Enhancements
 
 The following enhancements were made for the {{site.productname}} 5.8 release.
 
-### Enhancement name
+### Improved keyboard navigation for the inline boundary feature
 
-### Additional enhancements
+Users can now escape inline elements (such as links and comments) using the `Home`, `End`, `PageUp`, and `PageDown` keys. Due to their unique behavior, `PageUp` and `PageDown` are not available in macOS.
 
-{{site.productname}} 5.8 introduces the following minor enhancements:
+For information on inline boundaries, see: [Content appearance options - Inline boundaries]({{site.baseurl}}/configure/content-appearance/#inline_boundaries).
 
-- changelog
+### Additional enhancements
 
-## Accompanying Premium Plugin changes
+{{site.productname}} 5.8 introduces the following minor enhancements:
 
-The following premium plugin updates were released alongside {{site.productname}} 5.8.
+- Updated the `formatter.matchFormat` API to support matching formats with variables in the `classes` property.
+- Added HTML5 `audio` and `video` elements to the default alignment formats.
+- Added support for alpha list numbering to the list properties dialog.
```

Comment:
Should this be here, as I thought we were keeping these out of release notes for now?

Reply:
nope. Just not calling it out.